addon/components/bs-modal.js

```diff
@@ -193,6 +193,18 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+  /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using `true` and
+   *  `modal-fullscreen-[x]-down` class when using BS breakpoints
+   *   ([x] = 'sm', 'md', 'lg', 'xl', 'xxl').
```

Comment:
Would be great if you could add the link to the Bootstrap documentation for clarity: https://getbootstrap.com/docs/5.1/components/modal/#fullscreen-modal

Reply:
Fixed